releases/boltons/0.6.4/meta.yaml

```diff
@@ -0,0 +1,52 @@
+package:
+  name: boltons
+  version: {{ environ['GIT_DESCRIBE_TAG'] }}.post{{ environ['GIT_DESCRIBE_NUMBER'] }}

```

Comment:
remove the `post` bit of this.  As it is it will get a version like `0.6.4.post0` which while accurate is not super friendly.